requirements/base.txt

```diff
@@ -13,5 +13,5 @@ dateparser~=0.7
 python-dateutil~=2.6, <2.8.1
 requests==2.23.0
 serverlessrepo==0.1.9
-aws_lambda_builders==1.0.0
+aws_lambda_builders==1.1.0
```

Comment:
You need to update requirements file.

Reply:
Check https://github.com/aws/aws-sam-cli/blob/develop/Makefile#L51

Reply:
updated